### DISCUSSION TITLE
Citar at Emacs Conference 2021

### DISCUSSION THREAD

**Discussion by bdarcus** [emacs-citar/citar#446] [General]
There's a pretty cool talk by @rka97 at this year's Emacs Conference that features citar (starting around the 3:30 mark, and then again later on)!

https://emacsconf.org/2021/talks/research/

Nice work @rka97!

Note that the default citar note function now will add `ROAD_REFS` and id to org-roam notes.

**Comment by rka97** [comment]
Hi @bdarcus thanks so much for the package and the shout out :) it's great that we have these tools that integrate quite well together!

**Reply by bdarcus** [reply]
Sure thing; I subsequently added a link to it on the README.

**Reply by bdarcus** [reply]
I was actually (pleasantly) surprised citar was included, though, since it's not mentioned in the description/summary.

**Reply by rka97** [reply]
Yeah citar was very saliently used throughout the workflow so I forgot to write it out explicitly. Sorry :-)
